plugins/CoreHome/templates/_menu.twig

```diff
@@ -1,4 +1,4 @@
-{% macro menu(menu, anchorlink, cssClass, currentModule, currentAction, collapsible) %}
+{% macro menu(menu, anchorlink, cssClass, currentModule, currentAction, collapsible, hasSomeAdminAccess) %}
```

Comment:
I guess a check for super user permission (isSuperUser) would have been better here. There is no value in adding the new component for admin users, as the plugin menu is only rendered for super users and the new API method is also only available for superusers.

Reply:
Good point @sgiehl, created a tiny [follow up PR](https://github.com/matomo-org/matomo/pull/23654) for that if you can have a look.